### ISSUE TITLE
Undoing changes does not scroll us back to the changed area

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/handsontable#6402]
### Description

When we delete some cells, and use `undo` after being scrolled somewhere else we get the undo working. However, viewport does not move.
I would expect to be scrolled back to the area where the cells were returned. 

### Steps to reproduce

1. Go to https://handsontable.com/docs/7.2.2/demo-scrolling.html
2. Select a bunch of cells, let's say `B3:D10`
3. Click <kbd>DELETE</kbd>
4. Scroll table to see row `50`
6. Hit <kbd>CTRL/CMD</kbd> + <kbd>Z</kbd>

![GIF](https://user-images.githubusercontent.com/10757813/67772959-3cc4a900-fa5b-11e9-93fb-a60745e85c22.gif)

### Demo

https://jsfiddle.net/handsoncode/8ffpsqt6/

### Your environment
* Handsontable version: 7.2.2
_Does not seem to be version related_
* Browser Name and version: Chrome 77
_Does not seem to be browser-related_
* Operating System: Windows 10

**Comment by aninde** [comment]
@AMBudnik the expected result you described actually appears in Excel, but not in Google Spreadsheet. So is this really a bug? Topic for discussion.

I made reproduction in Google spreadsheet. After undoing changes I had to manually scrolled back.
![Screen Recording 2019-10-30 at 10 52 59 2019-10-30 10_55_24](https://user-images.githubusercontent.com/30186818/67847850-dcd60d00-fb03-11e9-8033-dce49de3005e.gif)

**Comment by AMBudnik** [comment]
Works the same in `8.0.0-beta2-rev21` https://jsfiddle.net/hvz3cyk6/

**Comment by krzysztofspilka** [comment]
@AMBudnik good catch but it’s definitely not a big. Let’s remove the label and discuss the desired behavior later on.

**Comment by pnowak** [comment]
This issue is still replicable using v8.0.0. Here's an updated demo - https://jsfiddle.net/whxntd2r/
